src/UserApp.ts

```diff
@@ -0,0 +1,173 @@
+import { DivsStructure } from "../utils/HtmlStructures"
+import { $, render, toLCaseIncludes, toArr } from "./variables/functions"
+
+
+// Enum with members representing buttons on the top of the app
+export enum CurrentButton {
+    Search,
+    Add,
+    Remove,
+    ShowAll
+}
+// User class
+export class User {
+    name: string
+    surname: string
+    dateOfBirth: number
+
+    constructor( name: string, surname: string, dateOfBirth: number ) {
+        this.name = name
+        this.surname = surname
+        this.dateOfBirth = dateOfBirth
+    }
+
+}
+// State class
+class UserAppState {
+    // Current button type
+    btnState: CurrentButton
+    // Array of found users
+    foundUsers: Array<User> = []
+    // Check if submit button clicked
+    ifSbmtClicked: boolean = false
+    // Check if user's input is right
+    ifExecutedRight: boolean
+    // Array of users
+    users: Array<User> = []
+    // Removed users
+    removedUser = []
```

Comment:
You do not annotate the type of this parameter. Please do.

Also please supply **more** comments/documentation regarding how this array is used. This will come very handy later for you and others.